### DISCUSSION TITLE
Is this plugin have protection for new version items?

### DISCUSSION THREAD

**Discussion by EMREOYUN** [Moneylover3246/Crossplay#22] [General]
As your plugin states, all 1.4.x.x clients can join to server but newest versions of the game have new items that players can join with it. If this happens, old clients can't respond this properly. Is your plugin have protection avout this issue?

**Comment by Moneylover3246** [comment]
Yes it does. The plugin patches newer version items, projectiles, buffs, etc. preventing crashes

**Comment by SaulloTschok** [comment]
hello I want to know how to put only version 1.4.3.2 up enter the server

**Reply by EMREOYUN** [reply]
I guess your question is not related to the plugin but let me answer: I'm using a load balancer to patch version code which allows upper versions can enter.
